frontend/src/component/changeRequest/DraftBanner/DraftBanner.tsx

```diff
@@ -75,16 +78,16 @@ const StickyBanner = styled(Box)(({ theme }) => ({
 
 export const DraftBanner: VFC<IDraftBannerProps> = ({ project }) => {
     const [isSidebarOpen, setIsSidebarOpen] = useState(false);
-    const { draft, loading } = usePendingChangeRequests(project);
+    const { data, loading } = usePendingChangeRequests(project);
 
-    if ((!loading && !draft) || draft?.length === 0) {
+    if ((!loading && !data) || data?.length === 0) {
         return null;
     }
 
     return (
         <StickyBanner>
-            {draft &&
-                draft
+            {data &&
```

Comment:
This will work, since it's an array and a truthy value. But && can be an error prone pattern: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx